flask_app/views.py

```diff
@@ -24,7 +24,9 @@ def predict():
         start_time = time.time()
 
         model = model_store.get('stargan')
-
+        
+        logging.getLogger().AddINFO(str(req))
```

Comment:
able to break down more specifically what's in the req? Sew whether we can log the specifics